app.js

```diff
@@ -73,6 +73,19 @@ async function integrityMetadata(text, algorithm) {
   return `${algorithm}-${base64string}`;
 }
 
+function checkBrowserDependency(url) {
+  const URL_LIST = [
+    'fonts.googleapis.com'
+  ];
+  const [, domain] = url.match(/^https?:\/\/([^/?#]+)(?:[/?#]|$)/i);
+
+  if (URL_LIST.find((e) => e === domain)) {
+    return [true, domain];
+  }
```

Comment:
Please use the bulit-in `URL()`  parser API. E.g.,
```js
let hostname = new URL(url).hostname;
if URL_LIST.includes(hostname) { 
// ...
```

Reply:
fixed